addons/spreadsheet/static/src/global_filters/components/boolean_multi_selector/boolean_multi_selector.js

```diff
@@ -0,0 +1,54 @@
+import { Component } from "@odoo/owl";
+import { TagsList } from "@web/core/tags_list/tags_list";
+import { AutoComplete } from "@web/core/autocomplete/autocomplete";
+import { _t } from "@web/core/l10n/translation";
+
+function toBoolean(value) {
+    return value === "true";
+}
+
+export class BooleanMultiSelector extends Component {
+    static template = "spreadsheet.BooleanMultiSelector";
+    static components = {
+        TagsList,
+        AutoComplete,
+    };
+
+    static props = {
+        selectedValues: Array,
+        update: Function,
+        placeholder: { type: String, optional: true },
+    };
+
+    onSelect({ value }) {
+        this.props.update([...this.props.selectedValues, toBoolean(value)]);
+    }
+
+    get placeholder() {
+        return this.props.selectedValues.length ? "" : this.props.placeholder;
+    }
+
+    get tags() {
+        return this.props.selectedValues.map((value) => ({
+            id: `${value}`,
+            text: value ? _t("Is set") : _t("Is not set"),
+            onDelete: () => {
+                this.props.update(this.props.selectedValues.filter((v) => v !== value));
+            },
+        }));
+    }
+
+    get sources() {
+        const options = [
+            { value: "true", label: _t("Is set") },
+            { value: "false", label: _t("Is not set") },
+        ];
```

Comment:
I know it's not big, but we could define this as static attribute on this component, and reuse it in `get tags`